embuilder.py

```diff
@@ -33,7 +33,6 @@
         libcxx_noexcept
         libcxxabi
         gl
-        struct_info
```

Comment:
For some reason I feel a bit dirty about this, trying to think up of a usage scenario where this would fail for the user, but not quite sure.

One issue is that the test is deleting `emscripten_root/src/struct_info.json`, which on Windows can be under c:\Program Files, which has read-only access, so the test doesn't run on such an installation, but not sure if that's an issue, since users can ignore running the tests, and mostly devs run the tests (and they won't install to Program Files, but will use git). Though this will be the first test that will write under `src/` and can fail like this I think.